src/rgw/rgw_op.cc

```diff
@@ -1648,15 +1649,26 @@ void RGWGetObj::execute()
   }
   /* end gettorrent */
 
+  // whether skip decompress to get object keeping compressed or not
```

Comment:
@ZVampirEM77 I believe It would be even better if we start comment with capital letter something as:
// Whether skip decompress to get object keeping compressed or not

Also Is there a need for space between `//` and `whether`